configs/STELLARF7V2/config.h

```diff
@@ -118,7 +118,6 @@
 #define FLASH_SPI_INSTANCE SPI3
 #define GYRO_1_SPI_INSTANCE SPI1
 #define GYRO_1_ALIGN CW180_DEG
-#define GYRO_1_ALIGN_PITCH 1800
```

Comment:
Wrong

Reply:
Here I'm confused why this is valid.

Reply:
180 pitch is cw0_flip

Reply:
Sorry for ignorance - as you stated settings are mutual exclusive - in my understanding when using `GYRO_1_ALIGN CW180_DEG` custom definition should be unavailable

![image](https://github.com/user-attachments/assets/50de2656-b55b-4e34-9e44-cd5e9c0fd239)

Reply:
I'm talking about config.h - it should use either GYRO_1_ALIGN CW180_DEG or #define GYRO_1_ALIGN_YAW 1800. In this case, ALIGN is CW180_DEG but custom alignment is GYRO_1_ALIGN_PITCH 1800. This does not match, there is some problem with config.